tracer/src/Datadog.Trace/Span.cs

```diff
@@ -459,5 +460,21 @@ internal void SetDuration(TimeSpan duration)
         {
             Duration = duration;
         }
+
+        /// <summary>
+        /// Sets a DataStreams checkpoint and adds pathway tag to the span
+        /// </summary>
+        /// <param name="manager">The <see cref="DataStreamsManager"/> to use</param>
+        /// <param name="edgeTags">The edge tags for this checkpoint. NOTE: These MUST be sorted alphabetically</param>
+        internal void SetDataStreamsCheckpoint(DataStreamsManager manager, string[] edgeTags)
+        {
+            Context.SetCheckpoint(manager, edgeTags);
+
+            var hash = Context.PathwayContext?.Hash.Value ?? 0;
+            if (hash != 0)
+            {
+                SetTag("pathway.hash", hash.ToString());
```

Comment:
Is the has meant to be encoded as a decimal string (as opposed to hexadecimal?). We have other values like trace id where in retrospect we should have encoded as hexadecimal because now it's a pain to change from 64 to 128 bits.

Alternatively, if it's a numeric tag, should it be set on the span's `metrics` map which contains doubles instead of the `meta` tag which requires strings?

Reply:
This tag  is a `FNV 64` hash of the path message took. I can't imagine a situation when 64bit will not be enough for our use case. It's passed as decimal string to make things consistent with other implementations (i.e. no unsigned values in Java).

Reply:
Thanks for clarifying! I will resolve this comment.